### ISSUE TITLE
Installing a Server-side subscription fails

### ISSUE THREAD

**Issue by akoenig** [graphcool/prisma#1922]
# Bug Report

**Current behavior**

When I define a server-side subscription query in my `prisma.yml` and try to `prisma deploy` it to my `local`, cluster I receive the following response:

```
ERROR: Whoops. Looks like an internal server error. Search your cluster logs for request ID: cluster:cluster:cjdq6pu0o001d0143keb01hmj

{
  "data": {
    "deploy": null
  },
  "errors": [
    {
      "message": "Whoops. Looks like an internal server error. Search your cluster logs for request ID: cluster:cluster:cjdq6pu0o001d0143keb01hmj",
      "path": [
        "deploy"
      ],
      "locations": [
        {
          "line": 2,
          "column": 9
        }
      ],
      "requestId": "cluster:cluster:cjdq6pu0o001d0143keb01hmj"
    }
  ],
  "status": 200
}
```

This is from the cluster logs: https://gist.github.com/akoenig/e247bd79748c720007db1390a205f7b1

I'm using `1.3.0-beta.1`

My `prisma.yml`:

```yml
service: my-service

stage: ${env:PRISMA_STAGE}

secret: ${env:PRISMA_SECRET}

datamodel: ...

cluster: ${env:PRISMA_CLUSTER}

subscriptions:
  userChangedFirstName:
    webhook:
      url: https://server-side-subscription.localtunnel.me/
      headers:
        Content-Type: application/json
        Authorization: Bearer foobar
    query: |
      {
        profile(where: {
          mutation_in: [UPDATED]
        }) {
          node {
            firstName
          }
        }
      }

seed:
  import: utils/seed.graphql

```

**Expected behavior?**

The server-side subscription should be deployed 😀

**Comment by marktani** [comment]
Thanks, I can confirm! @divyenduz also prepared a reproduction here: https://github.com/divyenduz/server-side-subscriptions

We'll look into this soon.

**Comment by PatrickStrz** [comment]
+1

**Comment by marktani** [comment]
We are investigating this issue. Until then, a workaround is to manually trigger an endpoint with the required data from a mutation resolver in your application server.

**Comment by akoenig** [comment]
@marktani Thanks for investigating this issue. :) Another workaround is to implement an own separate service that subscribes via WebSockets natively and sends the respective HTTP requests respectively.

**Comment by rohitghatol** [comment]
Which version of prisma has this fix?

**Comment by akoenig** [comment]
@rohitghatol I guess you should see the change in `1.4-beta-16` (Docker image), because there is no stable release with this commit yet. You can switch to the unstable channel via:

  * `npm i prisma@beta`
  * `prisma local upgrade`

**Comment by marktani** [comment]
Thanks @akoenig. I'll add that information to the README soon 🙂

**Comment by akoenig** [comment]
I just validated if the error persists in the current `1.4-beta` image. When registering the following subscription

```graphql
subscriptions:
  userChangedFirstName:
    webhook:
      url: http://192.168.0.10:9999/
      headers:
        Content-Type: application/json
    query: |
      {
        user(where: {
          mutation_in: [UPDATED]
        }) {
          node {
            profile {
              firstName
            }
          }
        }
      }
```

I get a `The provided query doesn't include any known model name.` error, when performing a `prisma deploy`. The subscription works in Playground though. Strange :/

**Comment by sorenbs** [comment]
Thanks @akoenig 

@do4gr will take a look at this again tomorrow.

**Comment by marktani** [comment]
This looks like this one: #1649.
Did you try to first deploy the data model, and then the subscription, @akoenig?

**Comment by marktani** [comment]
This is now fixed and available in `1.4.0` 🙂 Thanks a lot for your help with this, everyone 👌
